solidity/contracts/ValidatorRegistration.sol

```diff
@@ -1,23 +1,113 @@
+// Adapted from: https://github.com/prysmaticlabs/prysm/blob/master/contracts/validator-registration-contract/validator_registration.sol
 pragma solidity ^0.4.23;
```

Comment:
I think we should be using ^0.5

Reply:
@noot The latest stable truffle suite (`v4.1.15`) includes solc-js `v0.4.25`, so automatic testing doesn't allow compiler versions `^0.5`.

Reply:
truffle 5.0.0 (w/ solc 0.5.0) is available https://www.npmjs.com/package/truffle

Reply:
Hm. I see that on NPM but when I tried to update last night it just bumped me to 0.4.15. I’ll try to troubleshoot that. Thanks y’all @noot @ansermino

Reply:
@noot @ansermino PR updated

Reply:
Resolve this?

Reply:
Yes @GregTheGreek